deploy/90_rh-ssh.SSHD.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: cloudingress.managed.openshift.io/v1alpha1
+kind: SSHD
+metadata:
+  name: rh-ssh
+  namespace: "${{SSHD_NAMESPACE}}"
+spec:
+  dnsName: rh-ssh
+  allowedCIDRBlocks: "${{ALLOWED_CIDR_BLOCKS}}"
```

Comment:
@csheremeta how does this line up with the work you've done around this piece?

Reply:
This will work to pull in the IPs of the bastion nodes and the hive v4 shard NAT(s). If other IPs need to be included (i.e. the v3 hive node IPs, we'd need to pull those from the APIScheme object

Reply:
> Great! I think this should also be single curlies, @mbarnes

Ack.  So _all_ variables should be single curlies?